errbot/botplugin.py

```diff
@@ -266,6 +266,19 @@ def callback_message(self, message: Message) -> None:
         """
         pass
 
+    def callback_mention(self, message: Message, mentioned_people: Sequence[Identifier]) -> None:
+        """
+            Triggered if there are mentioned people in message.
```

Comment:
Oh we don't have the identity of the bot exposed to the plugin, we should do that also -_-, I can take this one separately if you wish.

Anyway, let's say we have...

```
self.bot_identifier = self._bot.bot_identifier
```

Then if the identity classes are correctly implemented `self.bot_identifier in people` should be true.

So ignore this for the moment, we will come back to it on another PR.